### ISSUE TITLE
Link ps-id in commit messages to git-ps website

### ISSUE THREAD

**Issue by ctsstc** [uptech/git-ps-rs#154]
It would be nice to help spread awareness for git-ps even more so, since simply searching for `ps-id` on google does not yield any helpful results. Maybe simply changing to `git-ps-id` could help, but you really do not get results unless it's just `git-ps` it seems.

One concern would be that changing this could break searching through commits if it doesn't stay the same, if someone was doing that. Maybe also having a standard ID in a hidden message could alleviate this.

**Comment by drewdeponte** [comment]
I am not exactly sure what you mean. Can you provide some examples of what exactly you are talking about.

**Comment by ctsstc** [comment]
Something like:

[ps-id](https://git-ps.sh/): abc-123-xyz-345


```
[ps-id](https://git-ps.sh/): abc-123-xyz-345
<!-- ps-id: abc-123-xyz-345 -->
```

I don't really like linking the text `ps-id` since it feels like that would go to the commit or link to something related to that ID, but it's the least intrusive compared to adding additional lines and/or branding.

Currently though when I'm working on a project where people have not yet heard of Git PS they have no clue or method to really figure out what it is, so I'm thinking additional branding/linking could help that.

This also feels like it works towards having a config for turning this off for people that would want to.

Edit: I guess if you wanted additional branding you could do it on a single line, or just add an extra line.

---

ps-id: abc-123-xyz-345  (created with [Git Patch Stack](https://git-ps.sh/))

---

ps-id: abc-123-xyz-345 
Learn more about [Git Patch Stack](https://git-ps.sh/)

---

ps-id: abc-123-xyz-345 
[Git Patch Stack](https://git-ps.sh/)

---

[Git Patch Stack](https://git-ps.sh/)
ps-id: abc-123-xyz-345

**Comment by drewdeponte** [comment]
Yeah, I think I almost prefer the explicit "Learn more about Git Patch Stack" under the ps-id
